src/protocol/runtime.rs

```diff
@@ -25,19 +26,90 @@ pub mod methods {
         pub properties: Vec<PropertyPreview>,
     }
 
-    #[derive(Deserialize, Debug, Clone)]
+    /// Object type
+    /// See https://chromedevtools.github.io/devtools-protocol/tot/Runtime#type-RemoteObject
+    #[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq)]
+    #[serde(rename_all = "lowercase")]
+    pub enum RemoteObjectType {
+        OBJECT,
+        FUNCTION,
+        UNDEFINED,
+        STRING,
+        NUMBER,
+        BOOLEAN,
+        SYMBOL,
+        BIGINT,
+    }
+
+    /// Object subtype hint. Specified for object type values only
+    /// See https://chromedevtools.github.io/devtools-protocol/tot/Runtime#type-RemoteObject
+    #[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq)]
+    #[serde(rename_all = "lowercase")]
+    pub enum RemoteObjectSubtype {
+        ARRAY,
+        NULL,
+        NODE,
+        REGEXP,
+        DATE,
+        MAP,
+        SET,
+        WEAKMAP,
+        WEAKSET,
+        ITERATOR,
+        GENERATOR,
+        ERROR,
+        PROXY,
+        PROMISE,
+        TYPEDARRAY,
+        ARRAYBUFFER,
+        DATAVIEW,
+    }
+
+    #[derive(Deserialize, Serialize, Debug, Clone, PartialEq)]
     #[serde(rename_all = "camelCase")]
     pub struct RemoteObject {
         #[serde(rename = "type")]
-        pub object_type: String,
-        pub subtype: Option<String>,
+        pub object_type: RemoteObjectType,
+        pub subtype: Option<RemoteObjectSubtype>,
         pub description: Option<String>,
         pub class_name: Option<String>,
         pub value: Option<serde_json::Value>,
         pub unserializable_value: Option<String>,
         pub preview: Option<ObjectPreview>,
     }
 
+    /// Experimental
+/// See https://chromedevtools.github.io/devtools-protocol/tot/Runtime#type-StackTraceId
+    #[derive(Debug, Clone, Deserialize, Serialize, PartialEq, Eq)]
+    pub struct StackTraceId {
+        id: String,
+        debugger_id: UniqueDebuggerId,
+    }
+
+    /// Call frames for assertions or error messages
+    /// See https://chromedevtools.github.io/devtools-protocol/tot/Runtime#type-StackTrace
+    #[derive(Debug, Clone, Deserialize, Serialize, PartialEq, Eq)]
+    #[serde(rename_all = "camelCase")]
+    pub struct StackTrace {
+        description: Option<String>,
+        call_frames: Vec<CallFrame>,
+        // parent: Option<StackTrace>,
+        /// Experimental
```

Comment:
Could you expand on the comment, giving some words about what is experimental and why.

Reply:
Yea, I'll write expand the comment about `parentId` and add a link to Chrome DevTools docs to this

https://chromedevtools.github.io/devtools-protocol/tot/Runtime#type-StackTraceId